oak_attestation_verification/src/lib.rs

```diff
@@ -15,6 +15,7 @@
 //
 
 #![cfg_attr(not(test), no_std)]
+#![feature(let_chains)]
```

Comment:
What is this feature?
Do we provide it? Or it comes from a dependency?

Reply:
It was needed to get code copied from oak_dice going. But we don't copy anymore, so this is gone.